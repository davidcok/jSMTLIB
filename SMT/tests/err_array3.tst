(set-logic QF_AX)
(declare-sort Bool 0)
(declare-sort Array 1)
(declare-fun select () Bool)
(declare-fun store () Bool)
