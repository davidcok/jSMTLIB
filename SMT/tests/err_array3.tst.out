success
(declare-sort Bool 0)
              ^^^^
(error "The identifier is already declared to be a sort: Bool")
(declare-sort Array 1)
              ^^^^^
(error "The identifier is already declared to be a sort: Array")
(declare-fun select () Bool)
             ^^^^^^
(error "Symbol select is already defined")
(declare-fun store () Bool)
             ^^^^^
(error "Symbol store is already defined")
